Clover/app/src/main/java/org/floens/chan/ui/view/MultiImageView.java

```diff
@@ -391,6 +408,21 @@ private void setVideoFile(final File file) {
             AndroidUtils.openIntent(intent);
 
             onModeLoaded(Mode.MOVIE, videoView);
+        } else if (ChanSettings.videoUseExoplayer.get()) {
+            exoVideoView = new PlayerView(getContext());
+            exoPlayer = ExoPlayerFactory.newSimpleInstance(getContext());
+            exoVideoView.setPlayer(exoPlayer);
+            DataSource.Factory dataSourceFactory = new DefaultDataSourceFactory(getContext(),
+                    Util.getUserAgent(getContext(),"Clover"));
```

Comment:
The `UserAgentProvider` could be used for this. You need to inject it, see other code that uses it.